a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -308,6 +309,7 @@ def execute(self, deltas):
                                                        nic.network_id)
                     network = self.network_driver.get_network(nic.network_id)
                     added_ports[amp_id].append(network)
+                    time.sleep(30)
```

Comment:
Why are we doing a time sleep here?

Reply:
Removed.

Reply:
Removing the time sleep will cause any further `System not ready` issues for subsequent tasks or not?

Reply:
I was not using correct vthunder image version.
By using 5.2.1 version image, not facing the issue.